src/utils/trusts.ts

```diff
@@ -291,8 +315,9 @@ const getLegalEntityTrustee = (
   appData: ApplicationData,
   trustId: string,
   trusteeId?: string,
+  isReview?: boolean
 ): TrustCorporate => {
-  const legalEntityTrustees = getLegalEntityBosInTrust(appData, trustId);
+  const legalEntityTrustees = getLegalEntityBosInTrust(appData, trustId, isReview);
 
   if (legalEntityTrustees.length === 0 || trusteeId === undefined) {
     return {} as TrustCorporate;
```

Comment:
{} is not a valid TrustCorporate as there's one non optional field: is_on_register_in_country_formed_in: YesNoResponse.

Reply:
should be returning undefined (or null which we've tended not to use)?

Reply:
existing code not touched by Christy's changes